hhs_hosp/delphi_hhs/run.py

```diff
@@ -139,6 +140,13 @@ def smooth_values(df, smoother):
     )
     return df
 
+def pop_proportion(df):
+    """Get the population-proportionate variants as the dataframe val."""
+    geo_mapper = GeoMapper()
```

Comment:
Pass the GeoMapper object in as an argument. GeoMapper has to read the crosswalk files from disk once for each instance, so ideally we only ever make one GeoMapper object.